lib/core/versions/latest/Encoder.ts

```diff
@@ -25,12 +26,19 @@ export default class Encoder {
   }
 
   /**
-   * Decodes the given Base64URL string into a string.
+   * Decodes the given input into the original string.
    */
   public static decodeAsString (encodedContent: string): string {
-    Encoder.validateBase64UrlString(encodedContent);
+    return Encoder.decodeBase64UrlAsString(encodedContent);
+  }
+
+  /**
+   * Decodes the given Base64URL string into the original string.
+   */
+  public static decodeBase64UrlAsString (input: string): string {
```

Comment:
Do we need this new function? The `decodeAsString` can be refactored to this as well right?

Reply:
The intention here is that this function will ALWAYS decode using Base64URL even if the default encoding scheme for the Encoder changes in future versions, to avoid bug creeping into decoding of things that's not tied directly to Sidetree spec, in this case, we are decoding JWS header, which is ALWAYS going to be base64URL.